extensions/wikia/GlobalWatchlist/GlobalWatchlist.md

```diff
@@ -0,0 +1,112 @@
+# Global Watchlist
```

Comment:
Didn't we decide to call this README.md so that it displays automatically on GitHub?